microsoft-edge/webview2/release-notes.md

```diff
@@ -27,6 +27,25 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
 To use a prerelease SDK along with a Microsoft Edge preview channel, navigate to [Switch to a preview channel to test upcoming APIs and features][SwitchToPreviewChannel].
 
 
+## 1.0.1018-prerelease
+
+Release Date: September 20, 2021
+
+[NuGet package for WebView2 SDK 1.0.1018-prerelease][NuGetGallery1.0.1018-prerelease]
+
+For full API compatibility, this prerelease version of the WebView2 SDK requires Microsoft Edge version 95.0.1018.0 or higher.
+
+### General
+
+#### Experimental Features
+*   Added [Media API][Webview2ReferenceWin32Icorewebview2experimental9ViewWebview2101018Prerelease] that enables developers to mute/unmute media within the WebView.
```

Comment:
@jasonstephen15 - I'm re-entering this "Outdated" comment, for visibility in the GitHub UI.

The destination page lacks the word 'media'. Will this "Media API" link be revised to go to a mid-page section that will have the word 'media'? The page that I arrive at doesn't have a clear relationship to the link text here.
The present .md file, rendered:
https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/release-notes?branch=pr-en-us-1468#experimental-features
(To test the Media API link, delete "review." from the 404 URL after click "Media API".)